src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDays = 7;
+  const shortTermDays = 3;
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
+  const price = carCostPerDay * days;
```

Comment:
I also recommend using a naming that will be more understandable.
```suggestion
  const basePrice = carCostPerDay * days;
```